node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/ApplicationSerializer.java

```diff
@@ -56,6 +57,7 @@ public class ApplicationSerializer {
     private static final String maxResourcesKey = "max";
     private static final String groupSizeKey = "groupSize";
     private static final String requiredKey = "required";
+    private static final String cloudAccountKey = "cloud-account";
```

Comment:
Nit: Make it camel-case like the others?

Reply:
Fixed